sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClientBuilder.java

```diff
@@ -351,6 +394,7 @@ public SecretClientBuilder pipeline(HttpPipeline pipeline) {
      *
      * @return The updated {@link SecretClientBuilder} object.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores some values in the `Configuration` object. Should we add a note like you did in other methods on this same class?